extensions/domain-analyzer/.gitignore

```diff
@@ -0,0 +1,27 @@
+# Dependencies
+node_modules/
+
+# Build output
+dist/
+
+# Cache
+.raycast/
+
+# Logs
+*.log
+
+# MacOS
+.DS_Store
+
+# IDE
+.vscode/
+.idea/
+
+# Environment variables
+.env
```

Comment:
Is all this needed? There shouldn't be a build output or env files, right? Please check the others too.